From 6b227e4e52596dab5e654234e13e7e50753db626 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Thu, 7 Jun 2018 19:59:46 +0300
Subject: [PATCH 09/32] Revert "extract_files: Add support for paths without
 system/"

This reverts commit 8b87b09ef7fc2deefc8b6b623c55da1c3dfa0342.
This will be addressed cleaner in the cleanup commit to follow.

Change-Id: Iec013bfafdbb334ab2657b10c433158002809b62
---
 build/tools/extract_utils.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/build/tools/extract_utils.sh b/build/tools/extract_utils.sh
index 4b919919..def2e0ad 100644
--- a/build/tools/extract_utils.sh
+++ b/build/tools/extract_utils.sh
@@ -961,8 +961,6 @@ function extract() {
             TARGET="$FROM"
             OUTPUT_DIR="$OUTPUT_DIR/rootfs"
             TMP_DIR="$TMP_DIR/rootfs"
-        elif [ -f "$SRC/$FILE" ] && [ "$SRC" != "adb" ]; then
-            TARGET="$FROM"
         else
             TARGET="system/$FROM"
             FILE="system/$FILE"
-- 
2.17.1

